.travis.yml

```diff
@@ -14,7 +14,7 @@ matrix:
         - ./install.py
         - ./install_test.py
 
-    - language: sh
+    - language: bash
```

Comment:
This could also be `shell` or `minimal`.

See: [https://docs.travis-ci.com/user/languages/minimal-and-generic/#aliases](https://docs.travis-ci.com/user/languages/minimal-and-generic/#aliases)

Personally, I liked `sh`, but I'm fine with `bash`, too. ;)

Reply:
I'm with you in this one. Liked `sh` more. 🙄